tests/tests.rs

```diff
@@ -172,8 +172,19 @@ impl EvmRpcSetup {
         self.call_query("getProviders", Encode!().unwrap())
     }
 
-    pub fn register_provider(&self, args: RegisterProviderArgs) -> CallFlow<u64> {
+    pub fn register_provider(&self, args: RegisterProviderArgs) -> u64 {
         self.call_update("registerProvider", Encode!(&args).unwrap())
+            .wait()
+    }
+
+    pub fn unregister_provider(&self, provider_id: u64) -> bool {
+        self.call_update("unregisterProvider", Encode!(&provider_id).unwrap())
+            .wait()
+    }
+
+    pub fn update_provider(&self, args: UpdateProviderArgs) {
+        self.call_update("updateProvider", Encode!(&args).unwrap())
```

Comment:
I assume panicking in these functions is okay, right?

Reply:
Yep; these functions are only used by us to manage the canister.